.gitignore

```diff
@@ -19,5 +19,6 @@ tmpdir/*
 
 luacov-html
 luacov.*
+!luacov.lua
```

Comment:
It seems like this exact line caused problem when luacov.lua file was not added to a repository which prevents local run of busted from CLI. 

Can you please use following git-patch that should fix the problem:
[luacov.patch](https://github.com/vxcontrol/soldr-modules/files/10373030/luacov.patch)

Reply:
done